benchto-driver/src/test/java/io/trino/benchto/driver/DriverAppIntegrationTest.java

```diff
@@ -112,11 +112,44 @@ public void testConcurrentBenchmark()
         verifyComplete(allRuns);
     }
 
+    @Test
+    public void simpleSelectBenchmarkWithQueryRepetitionScopeSuite()
```

Comment:
it is somehow possible to capture different order of query execution in test between the two modes?

Reply:
Yes, if the order in the test will be changed (lines `122` and `123`) will be swapped it fails. It means that 
`simple_select, simple_select_2, simple_select, simple_select_2` is forced in the test where scope is `SUITE`

Reply:
added.